opsys.ttl

```diff
@@ -182,8 +182,8 @@ source:
               <http://data.openlinksw.com/oplweb/opsys/universal-apple-macosx10.12-32#this> ,
               <http://data.openlinksw.com/oplweb/opsys/i686-apple-macosx10.13-32#this> ,
               <http://data.openlinksw.com/oplweb/opsys/x86_64-apple-macosx10.13-64#this> ,
-              <http://data.openlinksw.com/oplweb/opsys/universal-apple-macosx10.13-32#this> .
-
+              <http://data.openlinksw.com/oplweb/opsys/universal-apple-macosx10.13-32#this> ,
+	      <http://data.openlinksw.com/oplweb/opsys/universal-apple-macos11-64#this> .

```

Comment:
Given, and based on, existing listings, I think this list (and descriptions below) should also include `i686-apple-macosx10.14-32`, `x86_64-apple-macosx10.14-64`, and `x86_64-apple-macosx10.15-64`.

Possibly also `universal-apple-macosx10.14-32`, `x86_64-apple-macos11-64`, and `arm64-apple-macos11-64`.